.travis.yml

```diff
@@ -6,8 +6,6 @@ addons:
     - python-numpy
     - python-matplotlib
     - python-scipy
-    - python-pandas
-    - python-seaborn
```

Comment:
So which versions shall we support. On the first try pandas was indeed installed from ubuntu sources and since seaborn was not there pip did installed it. But the build fails because the version of pandas is ubuntu is so old that it loocks for attributes in the pandas dataframe that are not implemented yet.

```python-traceback
Unexpected failing examples:
../examples/plot_seaborn.py failed leaving traceback:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/sphinx_gallery-0.1.3.post1-py2.7.egg/sphinx_gallery/gen_rst.py", line 524, in execute_code_block
    exec(code_block, example_globals)
  File "<string>", line 10, in <module>
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/seaborn/utils.py", line 428, in load_dataset
    if df.iloc[-1].isnull().all():
  File "/usr/lib/pymodules/python2.7/pandas/core/frame.py", line 1437, in __getattr__
    (type(self).__name__, name))
AttributeError: 'DataFrame' object has no attribute 'iloc'
```

That is the reason I installed pandas and seaborn from pip, but it fails again probably now the matplotlib version is to old
```python-traceback
Unexpected failing examples:
../examples/plot_seaborn.py failed leaving traceback:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/sphinx_gallery-0.1.3.post1-py2.7.egg/sphinx_gallery/gen_rst.py", line 524, in execute_code_block
    exec(code_block, example_globals)
  File "<string>", line 15, in <module>
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/seaborn/axisgrid.py", line 1322, in map_diag
    diag_ax = ax._make_twin_axes(sharex=ax, frameon=False)
AttributeError: 'AxesSubplot' object has no attribute '_make_twin_axes'
```